ci-operator/step-registry/hypershift/optional-operators/OWNERS

```diff
@@ -0,0 +1,13 @@
+approvers:
+- csrwng
+- enxebre
+- sjenning
+- imain
```

Comment:
please drop imain and include Patryk-Stefanski for both reviewers approvers

Reply:
@enxebre Done, Thanks!